Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -370,7 +409,7 @@ public class FeaturevisorInstance {
                     enabled: stickyFeature.enabled,
                     sticky: stickyFeature)
 
-                  logger.debug("using sticky enabled", ["featureKey": featureKey]) // TODO: Log evaluation object. Make it encodable
+                  logger.debug("using sticky enabled", ["featureKey": evaluation])
```

Comment:
@PatrykPiwowarczyk, you should map **evaluation** to dictionary and pass it here. Now, under "featureKey" key you are passing the entire evaluation.

The goal is to have something like this:
`logger.debug("using sticky enabled", evaluation.asDictionary())`